R/experimentSpec.R

```diff
@@ -11,15 +11,14 @@
 #' @seealso [experimentSpecServer()] for the module server and a complete example.
 #' @export
 experimentSpecInput <- function(inputId,
-                                datasets,
+                                data,
                                 mae_name,
                                 label_experiments = "Select Experiment") {
   assert_string(inputId)
-  assert_r6(datasets)
+  assert_list(data)
   assert_string(mae_name, min.chars = 1L)
   assert_string(label_experiments, min.chars = 1L)
-
-  mae <- datasets$get_data(mae_name, filtered = FALSE)
+  mae <- data[[mae_name]]()
```

Comment:
The switch from non-filtered data to filtered data here should not have an impact as we are fetching the names of experiments only.